### PR TITLE
Add CVE re-scoring configuration to Logging stack images

### PR DESCRIPTION
**Pull request by vlvasilev**
**How to categorize this PR?**

/area logging
/kind enhancement

**What this PR does / why we need it**:
Add CVE re-scoring configuration to the Logging stack image, so we can automatically re-compute CVE scores for Logging related vulnerabilities.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: charts/images.yaml [hunk 1]
```diff
@@ -204,34 +204,107 @@ images:
...
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'public'
+      authentication_enforced: true
+      user_interaction: 'end-user'
+      confidentiality_requirement: 'none'
+      integrity_requirement: 'low'
+      availability_requirement: 'low'
+      comment: the network_exposure maybe should be considered protected because we restrict the request only to the push API.
```
Comment (JordanJordanov) [inline charts/images.yaml]: `network_exposure: 'public'` looks correct as long as there is interaction with peers from public internet. The restriction of the HTTP request method to `push` does not change the exposure.
<hr>

I have a question about  `user_interaction: 'end-user'`: 

> Declaring user interaction as end-user indicates that also untrained users can interact w/ the deployed service.

Do we really expect (and allow) untrained users to work with this component?
<hr>

Before removing the draft status of the PR, we shall update the comment labels replacing the questions with some explanations for our clasification decisions.

Comment (vlvasilev) [reply charts/images.yaml]: > Do we really expect (and allow) untrained users to work with this component?

The promtail agent is on the end-user node. As they can have admins who can ssh to their nodes they can manipulate the agent as they wish.

File: charts/images.yaml [hunk 2]
```diff
@@ -204,34 +204,107 @@ images:
...
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'private'
+      authentication_enforced: false
+      user_interaction: 'gardener-operator'
+      confidentiality_requirement: 'high'
+      integrity_requirement: 'low'
+      availability_requirement: 'low'
+      comment: this should be either the same as fluent-bit or just skipped because the image itself is only a carrier.
```
Comment (JordanJordanov) [inline charts/images.yaml]:
```suggestion
      confidentiality_requirement: 'none'
      integrity_requirement: 'none'
      availability_requirement: 'none'
      comment: no data is stored or processed by the installer
```

File: charts/images.yaml [hunk 3]
```diff
@@ -204,34 +204,107 @@ images:
...
   repository: grafana/loki
   tag: "2.2.1"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'private'
+      authentication_enforced: false
+      user_interaction: 'gardener-operator'
+      confidentiality_requirement: 'high'
+      integrity_requirement: 'low'
```
Comment (JordanJordanov) [inline charts/images.yaml]:
```suggestion
      integrity_requirement: 'high'
```

File: charts/images.yaml [hunk 4]
```diff
@@ -204,34 +204,107 @@ images:
...
   tag: "v0.47.0"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'private'
+      authentication_enforced: false
+      user_interaction: 'gardener-operator'
+      confidentiality_requirement: 'none'
+      integrity_requirement: 'low'
+      availability_requirement: 'none'
```
Comment (JordanJordanov) [inline charts/images.yaml]:
```suggestion
      integrity_requirement: 'high'
      availability_requirement: 'low'
```

File: charts/images.yaml (hunk 1 above)
Comment (JordanJordanov) [inline charts/images.yaml]:
```suggestion
      confidentiality_requirement: 'high'
      integrity_requirement: 'high'
      availability_requirement: 'low'
      comment: kube-rbac-proxy is an authentication proxy working with credentials 
```

File: charts/images.yaml [hunk 5]
```diff
@@ -204,34 +204,107 @@ images:
...
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'protected'
+      authentication_enforced: true
+      user_interaction: 'end-user'
+      confidentiality_requirement: 'high'
+      integrity_requirement: 'low'
+      availability_requirement: 'low'
+      comment: this container is only a carrier. network_exposure is 'protected' because it communicate only with kube-rbac-proxy container. But end-user can modified it.
```
Comment (JordanJordanov) [inline charts/images.yaml]:
```suggestion
      network_exposure: 'public'
      authentication_enforced: false
      user_interaction: 'end-user'
      confidentiality_requirement: 'high'
      integrity_requirement: 'high'
      availability_requirement: 'low'
      comment: network exposure is public due to the outbound connectivity, there is no exposed endpoint requiring auth
```

File: charts/images.yaml [hunk 6]
```diff
@@ -204,34 +204,107 @@ images:
...
   sourceRepository: github.com/gardener/logging
   repository: eu.gcr.io/gardener-project/gardener/telegraf-iptables
   tag: "v0.47.0"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      policy: skip
+      comment: >
+        telegraf is not accessible from outside k8s clusters and not
+        interacted with from other containers or other systems
```
Comment (JordanJordanov) [inline charts/images.yaml]:
```suggestion
      network_exposure: 'private'
      authentication_enforced: false
      user_interaction: 'gardener-operator'
      confidentiality_requirement: 'none'
      integrity_requirement: 'none'
      availability_requirement: 'none'
      comment: >
        telegraf is not accessible from outside the seed cluster and does not
        interact with confidential data
```

File: charts/images.yaml [hunk 7]
```diff
@@ -204,34 +204,107 @@ images:
...
   tag: "v0.47.0"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'private'
+      authentication_enforced: true
+      user_interaction: 'gardener-operator'
+      confidentiality_requirement: 'none'
+      integrity_requirement: 'none'
+      availability_requirement: 'none'
```
Comment (JordanJordanov) [inline charts/images.yaml]:
```suggestion
      authentication_enforced: false
      user_interaction: 'gardener-operator'
      confidentiality_requirement: 'high'
      integrity_requirement: 'high'
      availability_requirement: 'low'
```

**Review by JordanJordanov — CHANGES_REQUESTED** [review]
Team Review suggestions added

**Review by vpnachev — APPROVED** [review]
/lgtm

**Review by JordanJordanov — APPROVED** [review]
/lgtm
cc @nickytd

**Pull request merged**